src/features/currentTodo.ts

```diff
@@ -1,10 +1,17 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 const initialState = null as Todo | null;
 
 export const currentTodoSlice = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: {
+    open: (_, { payload }: PayloadAction<Todo>) => {
```

Comment:
in context of this slice `open` is not correct name for the action - it sets the current todo
So it should be `setCurrentTodo`